index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nimplay</title>
+    <link rel="shortcut icon" href="./Images/nimplayLogo.jpg">
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+  <header>           
+      <div class="bar2">
+          <div class="logo">
+            <a href="#" id="name">Nimrod Acosta</a> 
+          </div>
+          <nav class="burger">
+              <button class="normalb">
+                <img src="./Images/Union.png" class="union" alt="">                    
+              </button>
+            </nav>
+      </div>
+      
+
+  </header>
+  <section class="continer">
+      <h1 class="title">Hey There. I’m Nimrod</h1>
+      <h2 class="subtitle">I’am a Software Developer</h2>
+      <p class="paragraph">Encoding for life. <br> 
+        Web Solutions and Mobile Applications</p>
+      <ul class="social">
+          <li class="socialimg">
+            <img src="./Images/Github.png" alt="">
+          </li>
+          <li class="socialimg">
+            <img src="./Images/Icon-Linkedin.png" alt="">
+          </li>
+          <li class="socialimg">          
+            <img src="./Images/hand.png" alt="">
+          </li>
+          <li class="socialimg">
+            <img src="./Images/Icon-Twitter.png" alt="">
+          </li>
+          <li class="socialimg">
+            <img src="./Images/Icon-Medium.png" alt="">
+          </li>
+        </ul>                      
+  </section>    
```

Comment:
- [ ] Please make your design like the Figma design as you need to remove the spaces from the right, the left, and the bottom, also remove the space at the top of the header.
Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-05-26 23-29-15](https://user-images.githubusercontent.com/33752735/170583843-ffb032bb-bf5d-44f7-8ae0-dfe1f0f3b7c6.png) | ![Screenshot from 2022-05-26 23-02-35](https://user-images.githubusercontent.com/33752735/170583859-e68a502b-3650-4dc8-aa7f-bb9a8568efbc.png)

- [ ] Kindly ignore the `N`, `A`, and `O` icons.

Reply:
The leters n, a and d are not part of the layout, they are comment buttons...

Reply:
Yes I know I have mentioned that